galaxy/wrapper/computeMatrix.xml

```diff
@@ -118,6 +120,14 @@
                             label="Distance downstream of the region end position"
                             help="If the regions are genes, this would be the
                             distance downstream of the transcription end site."/>
+                        <param name="unscaled5prime" type="int" min="0" value="0"
```

Comment:
as a general note: if you use argument="--unscaled5prime" name will automatically calculated as argument.lstrip('-') and Galaxy will add (--unscaled5prime) to the help. This saves you a little bit of typing.